packages/typespec-vscode/test/unit/extension.test.ts

```diff
@@ -0,0 +1,16 @@
+import { assert, beforeAll, describe, it } from "vitest";
```

Comment:
add the scaffolding of unit test with a hello-world test to make it easy to add more test in the future.